src/main/kotlin/bootstrap.bzl

```diff
@@ -57,7 +57,7 @@ def kt_bootstrap_library(name, deps = [], neverlink_deps = [], srcs = [], visibi
 def kt_bootstrap_binary(
         name,
         main_class,
-        runtime_library,
+        runtime_deps,
```

Comment:
Renamed this to align better with what java_binary and the rest of the java rules use.